artemis-protocols/artemis-mqtt-protocol/pom.xml

```diff
@@ -47,29 +47,12 @@
       </dependency>
       <dependency>
          <groupId>io.netty</groupId>
-         <artifactId>netty-codec-mqtt</artifactId>
+         <artifactId>netty-all</artifactId>
       </dependency>
       <dependency>
          <groupId>junit</groupId>
          <artifactId>junit</artifactId>
       </dependency>
    </dependencies>
 
-   <build>
```

Comment:
@paulgallagher75 is going to test the build against MQTT example.  Should be fine though, the codec should be included in the lib.